x-pack/plugins/observability_solution/observability/public/navigation_tree.ts

```diff
@@ -402,6 +218,9 @@ export function createNavTree(pluginsStart: ObservabilityPublicPluginsStart) {
                   {
                     link: 'ml:settings',
                   },
+                  {
+                    link: 'ml:suppliedConfigurations',
```

Comment:
In this [PR](https://github.com/elastic/kibana/pull/191564) a new page for `Supplied Configurations` was added under the `Machine learning > Anomaly Detection` section.